docs/setup-ci.md

```diff
@@ -0,0 +1,104 @@
+
+Next you need to create a YAML file for GitHub Actions, the basic steps are:
+- Checkout code and setup the environment
+```yaml
+    - uses: actions/checkout@v2
+      name: Checkout Code
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14'
+
+    - name: Setup MSBuild
+      uses: microsoft/setup-msbuild@v1.0.2
+      with:
+        vs-version: 16.8
+
+    - name: Install node modules
+      run: yarn --frozen-lockfile
+
+    - name: yarn build
+      run:  yarn build
+```
+- Build and run the project
+```yaml
+    - name: Run Windows x64 release
+      run: npx react-native run-windows --arch x64 --release --no-packager --logging --deploy-from-layout
+```
+- Run tests
+```yaml
+    - name: Start Appium server
+      shell: powershell
+      run: Start-Process PowerShell -ArgumentList "yarn appium"
+
+    - name: Run tests
+      run: yarn test:windows
+```
+Check out the full [`react-native-webview` example](https://github.com/react-native-webview/react-native-webview/blob/master/.github/workflows/windows-ci.yml) as well as their [official example](https://github.blog/2019-08-08-github-actions-now-supports-ci-cd/) for more info.
+
+Add the YAML file to `.github\workflows\` and then commit. To know more about the YAML syntax, check out [Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions).
+
+> GitHub Actions should be enabled by default, if it's not enabled for some reason you can go to Settings->Actions tab of the repo to enable it (requires owner access).
+
+Now push your changes and the CI pipeline should be up and running.
+
+## Setting Up Pipeline for Signed Package Builds
+Certificates are used to sign RNW apps so that they can be installed locally or published to the Microsoft Store. This data should not be publically published, so we need to do extra steps if we wish to build/run signed RNW app packages through Github Actions.
+
+### Storing Certificates Securely
+There are a couple of options for where you can securing store your certificate information. Three options we have worked with are Github Secrets, Azure DevOps Secure Files, and Azure Key Vault. Github Secrets works well, when you are using Github Actions to run your pipeline. The latter two work well, if you are using Azure DevOps to run your pipeline. 
+
+Here is documentation on how to set up a Github Secret: [Github Secrets](https://docs.github.com/en/actions/reference/encrypted-secrets)
+Here is documentation on how to set up an Azure Secure File: [Azure Secure Files](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/secure-files?view=azure-devops)
+Here is documentation on how to set up an Azure Key Vault Secret: [Azure Key Vault Secrets](https://docs.microsoft.com/en-us/azure/key-vault/secrets/about-secrets)
```

Comment:
There are a several options where you can securely store your certificate information:

- [Github Secrets](https://docs.github.com/actions/reference/encrypted-secrets)
- [Azure DevOps Secure Files](https://docs.microsoft.com/azure/devops/pipelines/library/secure-files?view=azure-devops)
- [Azure Key Vault](https://docs.microsoft.com/azure/key-vault/secrets/about-secrets) 

Github Secrets works well, when you are using Github Actions to run your pipeline. The latter two work well, if you are using Azure DevOps to run your pipeline.

Reply:
Will update now!